### ISSUE TITLE
Remove the Raw Ironwood Materials item

### ISSUE THREAD

**Issue by Benimatic** [TeamTwilight/twilightforest#307]
A QOL change. Perhaps have the Liveroots smelt directly into Ironwood ingots, or for even more change, rename the Liveroot to ironwood root, and replace the ironwood ingots in recipes with the roots.

Perhaps also have the ironwood "ore" blocks drop 1-3 liveroot items instead of 1.

**Comment by Zygus42** [comment]
In my modpack I replaced the iron ingot in the recipe with copper - makes it an earlier age item.  Especially as I removed all iron ore from the Twilight Forest.  Was thinking of it like Feyiron - the copper/bronze-like material that the Fey use because they can't stand cold iron.  Also the Ironwood as it stands doesn't really make sense as it uses an iron bar and some harder to find substances to make and is only just marginally better than iron.
